icubam/predicu/__main__.py

```diff
@@ -19,31 +51,52 @@ def export_data(args):
   path = os.path.join(output_dir, filename)
   d = load_bedcounts(
     preprocess=True,
-    api_key=args.api_key,
-    max_date=args.max_date,
-    icubam_host=args.icubam_host,
-    spread_cum_jump_correction=args.spread_cum_jump_correction,
+    api_key=api_key,
+    max_date=max_date,
+    icubam_host=icubam_host,
+    spread_cum_jump_correction=spread_cum_jump_correction,
   )
   d.to_csv(path)
   logging.info("export DONE.")
 
 
+@cli.command(name="make_dashboard_plots", help="Make dasboard plots ")
+@click.option(
+  "--config", "-o", type=str, required=True, help="ICUBAM config file."
+)
+@click.option(
+  "--output-dir",
+  "-o",
+  type=str,
+  help=(
+    "Output directory where to store plots. If not provided "
+    "config.backoffice.extra_plots_dir is used."
+  )
+)
+def make_dasboard_plots(config, output_dir):
+  config = Config(config)
+  # Make sure DB path exists, to avoid creating an empty one
+  if not Path(config.db.sqlite_path).exists():
+    click.echo(f"Could not find DB at {config.db.sqlite_path}!", err=True)
+    sys.exit(1)
+
+  if output_dir is None:
+    output_dir = config.backoffice.extra_plots_dir
+
+  output_dir = Path(output_dir)
+  if not output_dir.exists():
+    if not output_dir.parent.exists():
+      click.echo(f"Output directory {output_dir}!", err=True)
+      sys.exit(1)
+    # otherwise necessary folder will be created by generate_plots
+
+  store_factory = create_store_factory_for_sqlite_db(config)
+  callback = AnalyticsCallback(output_dir, store_factory)
+  eventloop = asyncio.new_event_loop()
+  eventloop.run_until_complete(callback.generate_plots())
```

Comment:
It's a bit over-engineered but I just want to re-use the same callback as the one used to generate the plots with the backoffice server.